src/matrixroomhandler.ts

```diff
@@ -117,9 +117,9 @@ export class MatrixRoomHandler {
       return this.discord.ProcessMatrixStateEvent(event);
     } else if (event.type === "m.room.redaction" && context.rooms.remote) {
       return this.discord.ProcessMatrixRedact(event);
-    } else if (event.type === "m.room.message") {
-        log.verbose("Got m.room.message event");
-        if (event.content.body && event.content.body.startsWith("!discord")) {
+    } else if (event.type === "m.room.message" || event.type === "m.sticker") {
+        log.verbose("Got m.room.message / m.sticker event");
```

Comment:
That's just laaaazy, use ``Got ${event.type} event``

Reply:
ooops my brain somehow turned off